serveradmin/resources/management/commands/clear_migration_log.py

```diff
@@ -0,0 +1,50 @@
+"""Serveradmin - igvm_migration_log cleanup
+
+We are using igvm_migration_log for calculating the overall CPU usage of a
+Hypervisor. This is obsolete after the next prime time because the cpu_util_pct
+will be correct again. This task will get executed once in the morning to
+clean up the log entries from the previous day.
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+
+from adminapi.dataset import Query
+from adminapi.filters import Not, Empty
+
+
+class Command(BaseCommand):
+    help = 'Deletes igvm_migration_log entries from previous day'
+
+    def handle(self, *args, **options):
+        """Entry point for command
+
+        get all hypervisors which have an entry in igvm_migration_log
+        and delete all entries which are from the previous day.
+        """
+
+        all_hypervisors = Query({'igvm_migration_log': Not(Empty()),
+                                 'servertype': 'hypervisor'},
+                           ['igvm_migration_log', 'hostname'])
+
+        current_timestamp = datetime.utcnow()
```

Comment:
How about just `now` instead of `current_timestamp`, especially as it is not a `timestamp` but a datetime object?
```suggestion
        now = datetime.utcnow()
```